packages/webapp/__mocks__/reactMarkdownMock.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { ReactElement, ReactNode } from 'react';
```

Comment:
Why do we have 2 of these exact ones?

Reply:
Just a mock for each app. It is also done for svgr for example.